### ISSUE TITLE
Reference resources created by the community

### ISSUE THREAD

**Issue by sfmskywalker** [elsa-workflows/elsa-documentation#31]
- https://github.com/dammitjanet/ElsaSandbox
-

**Comment by Godaday** [comment]
If the community could provide a detailed documentation, the promotion and adoption of Elsa would see significant improvement

**Comment by sfmskywalker** [comment]
Yes, good, detailed documentation is vital. From your perspective, which areas would benefit the most from being documented? This might help prioritize what topics to do.
